src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -22,27 +22,28 @@ public Order add(Order order) {
             transaction = session.beginTransaction();
             session.save(order);
             transaction.commit();
+            return order;
         } catch (Exception ex) {
             if (transaction != null) {
                 transaction.rollback();
             }
+            throw new DataProcessingException("Can't insert new order into DB: " + order, ex);
         } finally {
             if (session != null) {
                 session.close();
             }
         }
-        return order;
     }
 
     @Override
     public List<Order> getByUser(User user) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             Query<Order> query = session.createQuery("from Order o "
```

Comment:
again, how many orders will this query return if you have `n` orders with `n` tickets in each order?
let's test it